cloud-controller-manager/do/loadbalancers.go

```diff
@@ -1321,6 +1331,20 @@ func getEnableBackendKeepalive(service *v1.Service) (bool, error) {
 	return enableBackendKeepalive, nil
 }
 
+func getHttpIdleTimeoutSeconds(service *v1.Service) (*uint64, error) {
+	httpIdleTimeoutSeconds, ok := service.Annotations[annDOHttpIdleTimeoutSeconds]
+	if !ok {
+		return nil, nil
+	}
+
+	httpIdleTimeout, err := strconv.ParseUint(httpIdleTimeoutSeconds, 10, 64)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse LB http_idle_timeout_seconds: %s", err)
```

Comment:
would this error be more helpful in debugging if we specified the full annotation key associated with the unparseable value?

Reply:
Added the full key like @anitgandhi suggested 🙏 

CC @khayyamsaleem  let me know if this looks okay